docs/content/users/quickstart.md

```diff
@@ -679,13 +679,13 @@ ddev launch /admin
 
 There are many ways to install Symfony, here are a few of them based on the [Symfony docs](https://symfony.com/doc/current/setup.html).
```

Comment:
```suggestion
```
Is that sentence really necessary? It is the purpose of quickstarts to provide one or two ways how to best get a certain CMS spun up. there is not really a need to state that there are several ways to install symfony imho

Reply:
This is probably left over from previous quickstart. OTOH... there *are* many ways, and it's more of a framework. So this may well be valid.